packages/tools/package.json

```diff
@@ -15,19 +15,25 @@
         "./*": "./src/*"
     },
     "scripts": {
-        "cli": "node ./src/index.js"
+        "cli": "ts-node ./src/index.ts"
```

Comment:
You can speed this up by running `ts-node -T` or `ts-node --transpileOnly`

Or even better `ts-node --swc`

Reply:
NICE!